<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1113</width>
    <height>649</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QListWidget" name="m_renderTypeList">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>Base color</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Metalness</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Roughness</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Normals</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Depth</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>OIT indices</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>OIT fragments count</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Final color</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Final metalness</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Final roughness</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Final normals</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Final depth</string>
         </property>
        </item>
       </widget>
       <widget class="QWidget" name="m_lightInfoWidget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>3</number>
         </property>
         <property name="leftMargin">
          <number>3</number>
         </property>
         <property name="topMargin">
          <number>3</number>
         </property>
         <property name="rightMargin">
          <number>3</number>
         </property>
         <property name="bottomMargin">
          <number>3</number>
         </property>
         <item>
          <layout class="QGridLayout" name="gridLayout" rowstretch="0,0" columnstretch="0,0">
           <property name="spacing">
            <number>3</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Shadow Mode</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="m_shadowModeBox">
             <item>
              <property name="text">
               <string>Disabled</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Opaque</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Opaque And Transparent</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Color</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Shadow Filter</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="m_shadowFilterBox">
             <item>
              <property name="text">
               <string>Point</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>PCF</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>VSM</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>73</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
     <widget class="QTextEdit" name="m_loggerTextEdit"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
